src/ResourceManager/Compute/Commands.Compute/packages.config

```diff
@@ -4,6 +4,11 @@
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Compute" version="16.3.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.KeyVault" version="2.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="15.1.0-preview" targetFramework="net452" />
```

Comment:
@sergey-shandar same comment about removing the added references since they should be referenced somewhere in common code